src/common/config.cpp

```diff
@@ -639,6 +644,9 @@ void save(const std::filesystem::path& path) {
 
     data["Settings"]["consoleLanguage"] = m_language;
 
+    // TODO Migration code, after a major release this should be removed.
+    data.at("GUI").as_table().erase("installDir");
+
```

Comment:
Ideally we should check if the string exists before we remove it.
Something like
```
// TODO Migration code, after a major release this should be removed.
auto old_game_install_dir =
        toml::find_or<std::string>(data.at("GUI"), "installDir", "doesn't exist");
    if (old_game_install_dir != "doesn't exist") {
        data.at("GUI").as_table().erase("installDir");
    }
```

Reply:
I thought the table does not throw an error when attempting to erase a key that doesn't exist?

Reply:
It doesn't but i guess its alright, since its going to be removed after some time. Ignore this then